✨ build buttons in components

mewiemers opened the pull request:
Fixes #13

add the first two buttons in storybook

mewiemers committed:
✨ build buttons in components
Fixes #13

add the first two buttons in storybook

mewiemers committed:
add variation for button in storybook

Deployment:
https://meet-albert-mewiemers-i-qql3gp.herokuapp.com/storybook/index.html?path=/story/common-button--traveling

mewiemers committed:
✨ add quizbutton in storybook

Deployment: https://meet-albert-mewiemers-i-qql3gp.herokuapp.com/storybook/

Sprint 1
 Part of Issue  build buttons in components #13

m-wiemers reviewed (approved):
We are building an app. And Melli shows how it works ⛑

phlgr commented on components/button/Button.module.css:
```diff
@@ -0,0 +1,15 @@
+  background-color: var(--button-color);
+  border: none;
+  font-family: "Gabriola", sans-serif;
+  width: 7rem;
+  height: 2rem;
```
Would not set a fixed `height` and `width`, the Button should get it's size by the content it is carrying.

phlgr commented on components/quizbutton/Quizbutton.module.css:
```diff
@@ -0,0 +1,33 @@
+.blue {
+  border-radius: 20px;
+  font-size: 1.25rem;
+  background-color: var(--quiz-button-color);
+  border: none;
+  font-family: "Gabriola", sans-serif;
+  width: 16rem;
+  height: 4.5rem;
+  box-shadow: 0.01px 7px 16px -2px rgba(0, 0, 0, 0.69),
+    0.01px 7px 16px -2px rgba(0, 0, 0, 0.69);
+}
+.red {
+  border-radius: 20px;
+  font-size: 1.25rem;
+  background-color: var(--quiz-button-false-color);
+  border: none;
+  font-family: "Gabriola", sans-serif;
+  width: 16rem;
+  height: 4.5rem;
+  box-shadow: 0.01px 7px 16px -2px rgba(0, 0, 0, 0.69),
+    0.01px 7px 16px -2px rgba(0, 0, 0, 0.69);
+}
+.green {
+  border-radius: 20px;
+  font-size: 1.25rem;
+  background-color: var(--quiz-button-true-color);
+  border: none;
+  font-family: "Gabriola", sans-serif;
+  width: 16rem;
+  height: 4.5rem;
+  box-shadow: 0.01px 7px 16px -2px rgba(0, 0, 0, 0.69),
+    0.01px 7px 16px -2px rgba(0, 0, 0, 0.69);
+}
```
You could create a new class standard which combines the standard attributes and remove the standard attributes from the color classes

phlgr commented on components/quizbutton/Quizbutton.tsx:
```diff
@@ -0,0 +1,18 @@
+  ...props
+}: QuizButtonProps) {
+  return (
+    <button className={styles[color]} {...props}>
```
```suggestion
    <button className={`${styles.standard} ${styles[color]}`} {...props}>
```

phlgr reviewed (changes requested):
Gj 🚀

mewiemers committed:
Update components/quizbutton/Quizbutton.tsx

Co-authored-by: Philipp <philippm.gartz@gmail.com>

mewiemers committed:
build buttons in components
Fixes #13
fix suggested changes

mewiemers committed:
xfix changes

Merge branch 'mewiemers/issue13' of github.com:mewiemers/meet-albert into mewiemers/issue13

mewiemers committed:
fix suggested change of width and height
